app/assets/javascripts/listings/components/listing/lotteryPreference.js.coffee

```diff
@@ -19,7 +20,10 @@ angular.module('dahlia.components')
       _.has(docSectionMap, @preference.preferenceName)
 
     @docSection = ->
-      docSectionMap[@preference.preferenceName]
+      if ListingIdentityService.isSale(@listing)
```

Comment:
what's going on with this function? I'm not super clear on what the goal is

Reply:
This is for anchors. The users for to checklist page, we want to direct them to related sections. For rental preferences or Homebuyers guide for sale listing. Same thing does the thing below, just the link is on the application form.

Reply:
hm. if the listing is sale they might still be looking for information on a preference. would this work on that case?